openai-java-core/src/main/kotlin/com/openai/services/blocking/ResponseService.kt

```diff
@@ -42,6 +44,17 @@ interface ResponseService {
         requestOptions: RequestOptions = RequestOptions.none(),
     ): Response
 
+    /** @see create */
```

Comment:
For each of these structured methods (here and for chat completions), it might be nice to add some info above the `@see create` explaining how this one differs